kroma-validator/guardian.go

```diff
@@ -288,7 +288,7 @@ func (g *Guardian) inspectOutput(outputIndex, fromBlock, toBlock *big.Int) {
 			retry := func() bool {
 				cCtx, cCancel := context.WithTimeout(g.ctx, g.cfg.NetworkTimeout)
 				defer cCancel()
-				isFinalized, err := g.IsOutputFinalized(cCtx, outputIndex)
+				isFinalized, err := g.isOutputFinalized(g.ctx, outputIndex)
```

Comment:
Why didn't you use `cCtx`? If you don't need to use it, delete the definition above.